root/work/WorkMerge.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import sortByEntityName
+  from '../static/scripts/common/utility/sortByEntityName';
+import EnterEdit from '../components/EnterEdit';
+import EnterEditNote from '../components/EnterEditNote';
+import FieldErrors from '../components/FieldErrors';
+import WorkList from '../components/list/WorkList';
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: MergeFormT,
+  +toMerge: $ReadOnlyArray<WorkT>,
+};
+
+const WorkMerge = ({$c, form, toMerge}: Props) => (
+  <Layout fullWidth title={l('Merge works')}>
+    <div id="content">
+      <h1>{l('Merge works')}</h1>
+      <p>
+        {l(`You are about to merge the following works into a single
+            work. Please select the work that you would like
+            the other works merged into::`)}
```

Comment:
Typo (not in the original TT) added in commit https://github.com/metabrainz/musicbrainz-server/pull/1431/commits/4627a92a506d8aae39c5f39b3b3ecaa17b0ce705, reverted after:

```suggestion
            the other works merged into:`)}
```